src/main.js

```diff
@@ -11,7 +11,8 @@ var background = document.querySelector('.background');
 
 var savedIdeas = [];
 
-// window.onload = retrieveMadeIdeaCards;
+window.onload = retrieveMadeIdeaCards;
```

Comment:
This pulls the array savedIdeas from localStorage and parses it out of being a string. It happens every time we refresh the page.